en/docs/adv_examples/bert.md

```diff
@@ -115,96 +116,84 @@ average speed: 0.556(sentences/sec)
+|run_squad.py|Run the SQuAD training|SQuAD|
+|run_squad_predict.py|Run the trained SQuAD model to predict.|SQuAD|
+|npy2json.py|Script required to overt OneFlow's prediction results to json.|SQuAD|
+|convert_tf_ckpt_to_of.py|Convert model from TensorFlow to OneFlow|BERT/SQuAD|
 
-* gpu_num_per_node： 每个节点上GPU的数目，OneFlow要求每个节点的GPU数目必须一致
 
-* node_num： 节点数目，即分布式训练时的主机数目
 
-* node_list： 节点列表，如果节点数大于1，则需要通过node_list指定节点列表，节点列表为字符串形式，采用逗号分隔，如`--node_num=2 --node_list="192.168.1.12,192.168.1.14`"
+### Options
+The script `run_pretraining.py` runs the pretraining and configured by command line options. You can run `run_pretraining.py --help` to see the options. The following is a detailed description of each option：
 
-* learning_rate： Learning rate
+* gpu_num_per_node: count of devices on each node which must be consistent on each machine
 
-* weight_decay_rate：设置权重衰减率
+* node_num: count of nodes, that is, the count of hosts in distributed system
 
-* batch_size_per_device： 分布式训练时每个设备上的batch大小
+* node_list: list of nodes. When thec count of nodes is more than one, we should spcifiy list of nodes by node_list. It's a string seperated by commans like `--node_num=2 --node_list="192.168.1.12,192.168.1.14"`
 
-* iter_num ITER_NUM： 训练的总轮数
+* learning_rate: learning rate
 
-* warmup_batches： 预热轮数，默认值为10000
+* weight_decay_rate: decay rate of weight
 
-* data_dir： OFRecord数据集的路径
+* batch_size_per_device: batch size on each device
 
-* data_part_num：OFRecord数据集目录下的数据文件数目
+* iter_num ITER_NUM: count of iterations
 
-* use_fp16： 是否使用fp16
+* warmup_batches: batches of warmup, default to 10000
 
-* use_boxing_v2： 是否使用boxing v2
+* data_dir: path to OFRecord dataset
 
-* loss_print_every_n_iter：训练中每隔多少轮打印一次训练信息（loss信息）
+* data_part_num: number of files in the folder of OFRecord dataset
 
-* model_save_every_n_iter： 训练中每隔多少轮保存一次模型
+* use_fp16: use float16 or not
 
-* model_save_dir： 模型存储路径
+* use_boxing_v2: use boxing v2 or not
 
-* save_last_snapshot：指定最后一轮训练完成后，模型保存路径
+* loss_print_every_n_iter: print loss every n iterations
 
-* model_load_dir：指定模型加载路径
+* model_save_every_n_iter: save the model every n iterations
 
-* log_dir LOG_DIR：指定日志路径
+* model_save_dir: path to save the model
 
-* seq_length： 指定BERT句子长度，默认值为512
+* save_last_snapshot: whether save the model when training is finished
 
-* max_predictions_per_seq： 默认值为80
+* model_load_dir: path to load the model
 
-* num_hidden_layers：隐藏层数目，默认值为24
+* log_dir LOG_DIR: specify the path of log
 
-* num_attention_heads： Attention头数目，默认值为16
+* seq_length: length of sequence, default to 512
 
-* max_position_embeddings：
+* max_predictions_per_seq: default to 80
 
-* type_vocab_size
+* num_hidden_layers: number of hidden layers, defaul to 24
 
-* vocab_size
+* num_attention_heads: number of attentoion heads，default to 16
 
-* attention_probs_dropout_prob
+### Use Wikipedia + BookCorpus dataset
+If it is necessary to carry out the pre-training of BERT from scratch, a large dataset should be used.
```

Comment:
If it is necessary to carry out the pretraining of BERT from scratch, a large dataset should be used.